docs/Installation/Installation.md

```diff
@@ -237,52 +237,50 @@ To build SpECTRE with Singularity you must:
-  shell script by adding `. /usr/local/Cellar/lmod/YOUR_VERSION_NUMBER/init/sh`
-  to your `.bash_profile`.
-* On Ubuntu, run `sudo apt-get install -y lmod` and, for Ubuntu 17.04, add
-  `. /etc/profile.d/lmod.sh` to your `.bashrc`. For Ubuntu 16.04, the correct
-  path to add is `. /usr/share/lmod/lmod/init/bash`.
-* On Arch Linux, run `yaourt -Sy lmod` and add `. /etc/profile.d/lmod.sh` to
-  your `.bashrc`,
-* On Fedora/RHEL, GNU Environment Modules comes out-of-the-box and works equally
-  well.
-* Instructions for other Linux distros are available online.
-
-To use modules with Spack, enable Spack's shell support by adding
-`. SPACK_DIR/share/spack/setup-env.sh` to your `.bash_profile` or `.bashrc`.
-
-Once you have Spack installed and configured with OpenSSL and LMod, you can
-install the SpECTRE dependencies using Spack. One way to install the
-dependencies is with a [Spack environment](https://spack.readthedocs.io/en/latest/environments.html):
+[Install Spack](https://spack.readthedocs.io/en/latest/getting_started.html) by
+cloning it into `SPACK_DIR` (a directory of your choice). For example, you can
+clone Spack into `SPECTRE_ROOT/support/spack`. Then, enable Spack's shell
+support with `source SPACK_DIR/share/spack/setup-env.sh`. Consider adding this
+line to your `.bash_profile` or `.bashrc`. Refer to [Spack's getting started
+guide](https://spack.readthedocs.io/en/latest/getting_started.html) for more
+information.
+
+Once you have Spack installed, one way to install the SpECTRE dependencies is
+with a [Spack environment](https://spack.readthedocs.io/en/latest/environments.html):
 
 \include support/DevEnvironments/spack.yaml
 
 You can also install the Spack packages listed in the environment file above
 with a plain `spack install` if you prefer.
 
-**Note**: Spack allows very flexible configurations and
-it is recommended you read the [documentation](https://spack.readthedocs.io) if
-you require features such as packages installed with different compilers.
-
-**Note**: On a Mac, you may need to `spack install
-yaml-cpp@develop~shared` (note that is a tilde and not a dash in front
-of shared) in order to force the building of the static libraries in
-order to avoid dynamic linking errors.
-
-## Building Charm++
+**Notes:**
+- Spack allows very flexible configurations and we recommended you read the
+  [documentation](https://spack.readthedocs.io) if you require features such as
+  packages installed with different compilers.
+- For security, it is good practice to make Spack [use the system's
+  OpenSSL](https://spack.readthedocs.io/en/latest/getting_started.html#openssl)
+  rather than allow it to install a new copy.
+- To avoid reinstalling lots of system-provided packages with Spack, use the
+  `spack external find` feature and the `--reuse` flag to `spack concretize` (or
+  `spack install`). You can also install some of the dependencies with your
+  system's package manager in advance, e.g., with `apt-get` or `brew`. If they
```

Comment:
I think Debian now recommends using `apt` instead of `apt-get` (though I don't know details)